src/commands/redis_cmd.cc

```diff
@@ -32,30 +32,30 @@
 #include <unordered_map>
```

Comment:
I'd recall naming this folder as `command`. What do you think?

Reply:
Do you mean singular `command` instead of plural `commands`?

Reply:
@torwig well. Since Kvrocks code is almost internal, that users talk to the server via wire protocol, I'm OK with either choice.